core/src/main/java/com/avast/bytes/BytesComparator.java

```diff
@@ -0,0 +1,17 @@
+package com.avast.bytes;
+
+import java.util.Comparator;
+
+/**
+ * Natural Bytes comparator that order Bytes as array of signed bytes.
+ *
+ * {-1} < {0}
+ * 0x80 < 0x00
+ */
+public class BytesComparator implements Comparator<Bytes> {
+
+    @Override
+    public int compare(Bytes a, Bytes b) {
+        return a.toReadOnlyByteBuffer().compareTo(b.toReadOnlyByteBuffer());
```

Comment:
Shouldn't this be implemented "manually"? Because this implementation means that if you have a list of `Bytes` and you want to sort them you will create `list.size()` instances of `ByteBuffer`.